app/executor.php

```diff
@@ -38,11 +40,11 @@
 // Fix error handling - done
 // Decide on logic for build and runtime containers names ( runtime-ID and build-ID) - done
 // Add size validators for the runtime IDs - done
+// Fix logging - done
+// Incorporate Matej's changes in the build stage ( moving of the tar file will be performed by the runtime and not the build stage ) - done
```

Comment:
should we remove this if this is done? May be if we need reference keep it as a PR description instead?

Reply:
done 👍